src/chain/chaininterface.rs

```diff
@@ -102,7 +143,8 @@ impl ChainWatchInterfaceUtil {
 		}
 	}
 
-	pub fn do_call_block_disconnected(&self, header: &BlockHeader) {
+	/// call listeners for disconnected blocks if they are still around
+	fn do_call_block_disconnected(&self, header: &BlockHeader) {
```

Comment:
Why make this non-pub?

Reply:
block_connected and block_disconnected meant to be called not these. I would probably inline them into those if I were not trying to make review easier.

Reply:
Maybe just delete this function and move it into block_disconnected instead of having the stub?